uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/absence-type-instance_test.js

```diff
@@ -0,0 +1,66 @@
+define([
+  'leave-absences/shared/models/instances/absence-type-instance',
+], function () {
+  'use strict'
+
+  describe('AbsenceTypeInstance', function () {
+    var AbsenceTypeInstance, ModelInstance;
+
+    beforeEach(module('leave-absences.models.instances'));
+
+    beforeEach(inject(function (_AbsenceTypeInstance_, _ModelInstance_) {
+      AbsenceTypeInstance = _AbsenceTypeInstance_;
+      ModelInstance = _ModelInstance_;
+    }));
+
+    it('inherits from ModelInstance', function () {
+      expect(_.functions(AbsenceTypeInstance)).toEqual(jasmine.arrayContaining(_.functions(ModelInstance)));
+    });
+
+    describe('init()', function () {
+      var instance;
+      var attributes = {
```

Comment:
You have already mocked data available. Just take one from there, why defining a new one here?